src/samples/HikingApp/HikingApp.MacCatalyst/ViewControllers/LoginViewController.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using UIKit;
+
+namespace HikingApp.MacCatalyst.ViewControllers
+{
+    public class LoginViewController : UIViewController
+    {
+        UIButton appleSignInButton = new UIButton(UIButtonType.System);
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+
+            View!.BackgroundColor = UIColor.White;
+
+            //  Create a button for Apple ID sign-in
```

Comment:
NIT: double space before comment 
```suggestion
            // Create a button for Apple ID sign-in
```